src/Uno.UI.Composition/Composition/CompositionVisualSurface.skia.cs

```diff
@@ -50,7 +50,12 @@ void ISkiaSurface.UpdateSurface(bool recreateSurface)
 					_surface.Canvas.Translate(-SourceOffset.X, -SourceOffset.Y);
 				}
 
+				bool? previousCompMode = Compositor.IsSoftwareRenderer;
+				Compositor.IsSoftwareRenderer = true;
+
 				SourceVisual.Draw(_drawingSession.Value);
+
+				Compositor.IsSoftwareRenderer = previousCompMode;
```

Comment:
nope, not a performance issue but the fact that some Skia features aren't available on software rendering (e.g. SkSl filters), and offscreen skia surfaces always use software rendering so we need to tell the compositor that we are on software rendering when rendering to an offscreen surface so that it doesn't try to use any unsupported features that aren't available on software rendering.